content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,240 @@
++++
+title = "Azure Data Explorer"
+layout = "scaler"
+availability = "v2.7+"
+maintainer = "Community"
+description = "Scale applications based on Azure Data Explorer query result."
+go_file = "azure_data_explorer_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-data-explorer` trigger that scales based on an Azure Data Explorer query result.
+
+```yaml
+triggers:
+- type: azure-data-explorer
+  metadata:
+    endpoint: "DATA_EXPLORER_ENDPOINT"
+    databaseName: "DATA_EXPLORER_DATABASE_NAME"
+    query: |
+      StormEvents
+      | summarize StormCount = count(), TypeOfStorms = dcount(EventType) by State
+      | top 1 by StormCount desc
+    threshold: "1000"
+    tenantId: AAD_APP_TENANT_ID # Can use TriggerAuthentication as well
+    clientId: AAD_APP_CLIENT_ID # Can use TriggerAuthentication as well
+    clientSecret: AAD_APP_SECRET # Can use TriggerAuthentication as well
```

Comment:
This looks like environment variables but I presume these should be concrete values, no?

Reply:
done